staging/src/k8s.io/client-go/examples/leader-election/README.md

```diff
@@ -7,15 +7,16 @@ This example demonstrates how to use the leader election package.
 Run the following three commands in separate terminals. Each terminal needs a unique `id`.
 
 ```bash
-# first terminal 
-go run *.go -kubeconfig=/my/config -logtostderr=true -id=1
+# first terminal
+go run main.go -kubeconfig=/path/to/kebuconfig -logtostderr=true -lease-lock-name=example -lease-lock-namespace=default -id=1
```

Comment:
> typo, should be kubeconfig (applies to all examples)

fixed.

> do the flag names require -- instead of -?

both `-` and `--` is ok to use. we can get more information from https://golang.org/pkg/flag/#hdr-Command_line_flag_syntax